Blog Post: Advancements in On-Chain Privacy (#411)
OpenSats/website: data/blog/developing-advancements-in-onchain-privacy.mdx

degrading the privacy—and thus security—of all users. Payjoin disrupts this
heuristic, which in turn improves the privacy characteristics of the whole
network. Even if only a small percentage of users use Payjoin as their default
payment mechanism, the common input ownership heuristic can't be trusted
anymore, making it a useless heuristic for transaction deanonymization. *"The
main problem is that,"* as Francis Pouliot, CEO of [Bull
Bitcoin](https://www.bullbitcoin.com/) pointed out, *"to make it happen, a lot of
open-source software infrastructure was missing."*

Version 1 of Payjoin ([Pay-to-EndPoint](https://en.bitcoin.it/wiki/PayJoin))
disrupted the CIOH assumption by modifying how transactions are constructed.
Instead of a straightforward sender-to-recipient payment, the recipient also
contributes one of their inputs to the transaction. This process reduces the
effectiveness of common tracking and surveillance methods in use today. However,
the challenge with this version was that it required real-time coordination,
meaning both sender and receiver had to be online at the same time, which is not
practical at scale.

> Thanks in part to [OpenSats
> funding](/blog/bitcoin-grants-july-2023#payjoin-dev-kit), Payjoin V2 ([BIP
> 77](https://github.com/bitcoin/bips/pull/1483)) launched in 2024, implementing a
> major improvement called asynchronous transactions. Due to this, Payjoin V2 no
> longer requires both sender and receiver to be online simultaneously, making
> private and secure transactions seamless for users. This enhancement makes it
> easier for mobile wallets and exchanges to integrate Payjoin without disrupting
> their existing user workflows, making privacy and security protections more
> accessible to a wider audience.

Reviewer: ```suggestion
Thanks in part to [OpenSats
funding](/blog/bitcoin-grants-july-2023#payjoin-dev-kit), Payjoin V2 ([BIP
77](https://github.com/bitcoin/bips/pull/1483)) launched in 2024, implementing a
major improvement called Async Payjoin. Because of this, Payjoin V2 no
longer requires both sender and receiver to be online simultaneously, making
private and secure transactions seamless for users. This enhancement makes it
easier for mobile wallets and exchanges to integrate Payjoin by default without disrupting
their existing user workflows, making privacy and security protections more
accessible to a wider user base.
```

- called "Async Payjoin" because you won't find "asynchronous transactions" in a search
- Because of (the action, implementation) rather than Due to (the noun, a specific spec)
- - by default because that links back to your opener
- user base rather than audience because the so-called 'audience' are economic actors, not just viewers

> In December 2024, Bull Bitcoin became the first bitcoin exchange to process
> Payjoin V2 withdrawals natively. This

Reviewer: ```suggestion
In December 2024, Bull Bitcoin Mobile became the first mobile wallet to enable
sending and receiving Async Payjoin. This
```

The exchange hasn't enabled deposits / withdrawals yet (2 weeks). The self-custody mobile wallet got the update.

bonds](https://jamdocs.org/glossary/#fidelity-bond) to resist [Sybil
attacks](https://en.wikipedia.org/wiki/Sybil_attack). Despite the protocol's
complexity, Coinswap prioritizes user-friendly workflows to make financial
privacy accessible.

The recent beta release of
[v0.1.0](https://github.com/citadel-tech/coinswap/milestone/1) marks a major
milestone towards making Coinswap use more prevalent. This testnet release aims
to demonstrate how swaps can function in real-world conditions without revealing
which participants exchanged UTXOs.

Looking ahead, the project plans to implement Taproot outputs, MuSig2 signatures
for smaller and more private transactions, and decentralized DNS discovery to
enable permissionless access to swap servers. The project is also exploring
cross-chain swaps with UTXO-based sidechains like Liquid.

By evolving alongside Bitcoin's broader ecosystem, Coinswap ensures that
financial privacy remains practical and widely available, empowering individuals
and businesses to transact freely without fear of surveillance.

### Silent Payments

Developed by OpenSats
[LTS](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors)

> grantee [Josi Baker](/blog/josi-baker-receives-opensats-lts-grant), Silent

Reviewer 2: Will verify with him in person tomorrow, but I thought his first name was Josie.

```suggestion
grantee [Josie Baker](/blog/josi-baker-receives-opensats-lts-grant), Silent
```

Reviewer 3: Good catch, thank you. (We went with Josi [in the past](https://opensats.org/blog/josi-baker-receives-opensats-lts-grant), but I just noticed that he put "josie" in his [GitHub profile](https://github.com/josibake).)

Reviewer 2: Confirmed with him: Josie

Reviewer 2: Thanks for updating! Would suggest updating s/Josi/Josie/ Baker in the blog post this article links to: https://opensats.org/blog/josi-baker-receives-opensats-lts-grant
